src/main/java/uk/gov/justice/hmpps/casenotes/services/CaseNoteService.java

```diff
@@ -128,29 +127,17 @@ private List<CaseNote> translateToDto(final Page<NomisCaseNote> pagedNotes, fina
             .collect(Collectors.toList());
     }
 
-    @SuppressWarnings("unchecked")
     private static List<CaseNote> sortByFieldName(
         final List<CaseNote> list,
         final String fieldName,
         final Sort.Direction direction
     ) {
-        try {
-            final var field = CaseNote.class.getDeclaredField(fieldName);
-            field.setAccessible(true);
-
-            return list.stream()
-                .sorted((first, second) -> {
-                    try {
-                        return direction == ASC ? ((Comparable<Object>) field.get(first)).compareTo(field.get(second))
-                            : ((Comparable<Object>) field.get(second)).compareTo(field.get(first));
-                    } catch (IllegalAccessException e) {
-                        throw new RuntimeException("Error", e);
-                    }
-                })
-                .collect(Collectors.toList());
-        } catch (final NoSuchFieldException e) {
-            return list;
-        }
+        final Comparator<CaseNote> compare = fieldName.equalsIgnoreCase("creationDateTime") ?
+            comparing(CaseNote::getCreationDateTime) : comparing(CaseNote::getOccurrenceDateTime);
```

Comment:
My interpretation of this change is the default ordering is now by occurrence date time whereas I think it used to default to natural ordering (the catch block). If that's the case, I'd probably switch it round and have it default to created descending